src/v2/System/Analytics/MNTN/MntnConversionPixel.tsx

```diff
@@ -12,7 +12,7 @@ const MntnConversionPixel: FC<MntnConversionPixelProps> = ({ path }) => {
     if (matchFound && !document.getElementById("mntn_conversion")) {
       script = document.createElement("script")
       script.id = "mntn_conversion"
-      script.src = src
+      script.text = text
```

Comment:
Interesting, perhaps we could update `useLoadScript` to check for `src` _or_ `text` (if one passes in `src`, use that, if one passes in `text`, use that. I bet that would solve the issue.